CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ All notable changes to this project will be documented in this file.
 - spark: Add HBase connector ([#878], [#882]).
 - hbase: hbase-entrypoint.sh script to start and gracefully stop services ([#898]).
 - tools: install yq command line tool for YAML manipulation ([#912]).
+- nifi: Add version 2.0.0 ([#917]).
```

Comment:
I didn't check, but was there a changelog entry for the M version in the unreleased section? Maybe it can be removed.

Reply:
Yeah it was not there, maybe because of the experimental (or simply forgotten)? IMHO i would leave it in?